src/deployment/efcore/tasks/README.md

```diff
@@ -1,83 +1,17 @@
-# EF Core Pipeline Steps
+# EF Core Deployment Steps
```

Comment:
can we have a script for running an EF bundle if we've got a build script for it?

Reply:
Will give this some more thought, there are differences between how you run the bundle on a linux/windows agent.

Reply:
Have added a script that has been tested on both windows and linux agents